AUTHORS.rst

```diff
@@ -11,3 +11,4 @@ Contributors
 ~~~~~~~~~~~~
 
 * Khaled Porlin - https://github.com/porlin72
+* [Agam Dua](https://github.com/agamdua)
```

Comment:
I like that both name and link are explicitly rendered which is used for PDF docs, etc... I dont have any strong preference either way